guides/v2.3/rest/tutorials/index.md

```diff
@@ -15,6 +15,8 @@ The REST tutorials provide an introduction to Magento web APIs. In general, the
 
 * The [**configurable product** tutorial]({{ page.baseurl }}/rest/tutorials/configurable-product/config-product-intro.html) helps you plan then create a configurable product and its component simple products.
 
+* The [**bulk API configurable product** tutorial]({{ page.baseurl }}/rest/tutorials/bulk-configurable-product/config-product-intro.html) demonstrates how to perform actions on a large scale; such as, creating multiple customers and products, changing prices across an inventory, and assigning large groups of products to a specific warehouse, all within a single call.
```

Comment:
This tutorial only shows how to perform one task: creating configurable products. Rewrite this paragraph.

Reply:
rewrote to be more specific about what the tutorial is actually demonstrating.